bih-cluster/docs/first-steps/episode-0.md

```diff
@@ -45,7 +45,7 @@ In preparation for our first steps tutorial series, we would like you to install
 In general the users on the cluster will manage their own software with the help of conda.
 If you haven't done so so far, please [follow the instructions in installing conda](../best-practice/software-installation-with-conda.md) first.
 The only premise is that you are able to [log into the cluster](../connecting/configure-ssh/linux.md).
-Make also sure that you are logged in to a computation node using ``.
+Make also sure that you are logged in to a computation node using `srun -p long --time 7-00 --mem=8G --ntasks=8 --pty bash -i`.
```

Comment:
Hi Sam,

can you please reduce the srun resources a little? Like medium partition with 1 or 2 cores should be enough for conda, shouldn't it?

Will merge this afterwards, thanks a lot!
Clemens

Reply:
Hi Clemens, 

Sure, as it's just for conda I've reduced the resources to

`srun -p medium --time 1-00 --mem=4G --ntasks=1 --pty bash -i`

Best wishes,
Sam